lib/result.js

```diff
@@ -58,11 +58,16 @@ export function createResult(
   if (code.search(/[\S]/) != -1) {
     switch (cellType) {
       case "markdown":
-        outputStore.appendOutput(convertMarkdownToOutput(code));
+        if (
+          globalOutputStore &&
+          atom.config.get("Hydrogen.showMarkdownInOutputArea")
```

Comment:
Why not just watch on `globalOutputStore` ?
Introducing another configuration only for this specific case sounds like too verbose to me.

Reply:
The only reason I decided to add a new configuration was because I did not want to change the existing behavior of the output dock. But you're right, it is a pretty specific use case and I can remove the `showMarkdownInOutputArea` setting from the configurations if you prefer.

Reply:
yeah let's remove that, that would make markdown cell rendering consistent with code evaluation, and that would be better, I think.

Reply:
Done. The changes to `lib/config.js` have been removed and the changes to `lib/result.js` has been updated to only depend on `globalOutputStore`.